pyshtools/meson.build

```diff
@@ -0,0 +1,51 @@
+python_sources = [
+  '__init__.py',
+  '_version.py',
+]
+
+pymod = import('python')
+python = pymod.find_installation(modules: ['numpy'])
```

Comment:
If you bumped the minimum required meson version up to 0.64.0 then you could pass `pure: false` as a kwarg to find_installation and not have to repeat it everywhere else (also not have to worry if you forget it).

Reply:
The minimum was pretty old; I can probably bump that up to 1.0.0 at this point.